CakePHP/Sniffs/NamingConventions/ValidClassBracketsSniff.php

```diff
@@ -19,27 +19,32 @@
  * Ensures curly brackets are on the same line as the Class declaration
```

Comment:
This sniff seems useless since PSR2 expects the curly bracket on next line of class declaration.